styles/index.js

```diff
@@ -1,4 +1,6 @@
+const base = require('./base')
+
 module.exports = {
-  light: require('./light'),
-  dark: require('./dark')
+  light: base + require('./light'),
+  dark: base + require('./dark'),
```

Comment:
I would prefer if the base styles were included in the theme's `index.js` instead of the shared one here. 

To me, this would seem easier to follow.

Reply:
After commenting this, I discovered why some of the SVG images weren't appearing correctly (see other comment), so I think this will have to be changed to fix that problem.